### ISSUE TITLE
Suitecrm helm deployment

### ISSUE THREAD

**Issue by nparfait** [bitnami/charts#4672]
**Which chart**:
suitecrm 9.0.1

**Describe the bug**
The suitecrm pod keeps restarting.
I have run this on AWS EKS and also locally on docker desktop k8s cluster.
application starts up and is accessible. the install script runs and i can log in. After a few minutes the pod is terminated (if i am logged into the pod i get a 
command terminated with exit code 137
kubectl describe pod shows
    Last State:     Terminated
      Reason:       Error
      Exit Code:    7
      Started:      Thu, 10 Dec 2020 17:10:37 +1100
      Finished:     Thu, 10 Dec 2020 17:14:28 +1100

**To Reproduce**
1. Install chart to k8s cluster.

**Expected behavior**
pod to not terminate

**Version of Helm and Kubernetes**:
kubernetes - v1.18.8

- Output of `helm version`:
```
version.BuildInfo{Version:"v3.1.2", GitCommit:"d878d4d45863e42fd5cff6743294a11d28a9abce", GitTreeState:"clean", GoVersion:"go1.13.8"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.8", GitCommit:"9f2892aab98fe339f3bd70e3c470144299398ace", GitTreeState:"clean", BuildDate:"2020-08-13T16:04:18Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by marcosbc** [comment]
Hi @nparfait, could you share the logs of your pod?

In any case, we have found a possible issue that could be happening when the SuiteCRM hostname (`suitecrmHost` value) is not properly set, and could cause a CrashLoopBackOff. We're working on releasing a fix for the container image and will keep you in the loop.

**Comment by marcosbc** [comment]
Hi @nparfait, we just released version 9.0.2 of the SuiteCRM chart which may fix your issues. It includes the SuiteCRM container version [`7.11.18-debian-10-r31`](https://github.com/bitnami/bitnami-docker-suitecrm/commit/0dfd7a80b64d189f791f524904241f6be9af15fa).

**Comment by nparfait** [comment]
This is fixed now. Thanks.
